test/box-luatest/gh_7394_memtx_mvcc_space_cnt_bug_test.lua

```diff
@@ -0,0 +1,38 @@
+local server = require('luatest.server')
```

Comment:
The test seem to belong to `engine-luatest/`, because it is memtx-specific. Otherwise looks good.

Reply:
I guess you are right, but we traditionally add memtx transaction manager tests to `box` suites, so I think we should stick to it. BTW, this test can actually be made multi-engine.